src/app/public/modules/numeric/numeric.pipe.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
 
 import {
   SkyLibResourcesService
-} from '@skyux/i18n';
+} from '@skyux/i18n/modules/i18n/lib-resources.service';
 
 import {
   SkyLibResourcesTestService
```

Comment:
Do you need the direct path for `SkyLibResourcesTestService` as well?

Reply:
Not really; since unit tests are run local to `@skyux/core`. The circular dependency only occurs in production. I'll change it here, though.